### PR TITLE
drop type specs in RecoJets

### PR DESCRIPTION
**Pull request by jeongeun**
#### PR description: 
Update the safer syntax for existing parameter :
-- drop type specifications where the original parameter exists.
-- move all parameter inside the clone

Instead of modifying parameters with full type specs, which can be interpreted as an insertion of a new parameter, it is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.
(The references were [PR#30700](https://github.com/cms-sw/cmssw/pull/30700), [PR#30827](https://github.com/cms-sw/cmssw/pull/30827), [PR#30947](https://github.com/cms-sw/cmssw/pull/30947))

In this PR, total 18 files updated.

-RecoJets/Configuration  5 files
-RecoJets/JetPlusTracks   1 file
-RecoJets/JetProducers   12 files

#### PR validation:
Event Content comparison check was also done and there is no change with these updates.
Tested in CMSSW_11_2_X, the basic test all passed in the [CMSSW PR instructions](https://cms-sw.github.io/PRWorkflow.html).

### CODE REVIEW

**Comment by slava77** [comment]
@cmsbuild please test

File: RecoJets/JetProducers/python/ak8PFJets_cfi.py [hunk 1]
```diff
@@ -61,9 +61,9 @@
 # Substructure algorithms
 ##############################################################################
 ak8PFJetsCHSFiltered = ak8PFJets.clone(
-    src = cms.InputTag("ak8PFJetsCHSConstituents", "constituents"),
+    src = "ak8PFJetsCHSConstituents:constituents",
     useFiltering = cms.bool(True),
-    nFilt = cms.int32(3),
+    nFilt = cms.double(3),
```
Comment (perrotta) [inline RecoJets/JetProducers/python/ak8PFJets_cfi.py:66]: `nFilt` is a `cms.int32`: it was correct, please restore

**Comment by perrotta** [comment]
please test

**Comment by perrotta** [comment]
please test

**Comment by perrotta** [comment]
+1
- Type specs dropped from cloned/modified reco modules as planned
- Jenkins tests pass

**Comment by qliphy** [comment]
+1

**Pull request merged**